content/en/docs/Overview/your-first-pipeline.md

```diff
@@ -19,106 +19,106 @@ You can start a pipeline manually, or you can configure it to be automatically t
+   ![Pipeline visual representation, with only a config added](/images/overview/your-first-pipeline/pipeline-config-only.png)
 
-Step 7: Before you test your pipeline, you may want to consider enabling or disabling the trigger via the checklist at the bottom.
+1. Select Bake from the different Types category.
 
-Step 8: Now we add our first stage: Baking an AMI. Click the add stage button in the visual representations section.
+   ![The stage type menu, selecting "Bake"](/images/overview/your-first-pipeline/add-bake-stage.png)
 
-![](/images/Image-2017-03-24-at-4.19.38-PM.png)
+1. Select Amazon from the Provider list (if multiple providers are configured), then the region you want to bake in. Enter the name of the package that was archived by the Jenkins job.
 
-Step 9: Select Bake from the different Types category.
+   - The package name should not include any version numbers. (e.g.: If your build produces a deb file named “myapp_1.27-h343”, you would want to enter “myapp” here.)
+   - If you would like to configure your own Base AMI under the Advanced Options, the Base OS configuration will be ignored.
 
-![](/images/Image-2017-03-24-at-4.20.02-PM.png)
+   ![Bake configuration for an AMI image](/images/overview/your-first-pipeline/bake-ami-config.png)
 
-Step 10: Select Amazon from the Provider list, then the region you want to bake in. Enter the name of the package that was archived by the Jenkins job.
+### Add a Deploy stage
 
-Note: The package name should not include any version numbers. (e.g.: If your build produces a deb file named “myapp_1.27-h343”, you would want to enter “myapp” here.)
-Note 2: If you would like to configure your own Base AMI under the Advanced Options, the Base OS configuration will be ignored.
+1. Now add a Deploy stage by clicking **{{< icon "plus-circle" >}} Add stage** again. In the Type category select Deploy. Deploy’s configuration settings should pop up on the screen.
 
-![](/images/Image-2017-03-24-at-4.26.08-PM.png)
+   ![Add a Deploy stage to the Pipeline](/images/overview/your-first-pipeline/add-deploy-stage.png)
 
-Step 11: Now add a Deploy stage by clicking Add Stage again. In the Type category select Deploy. Deploy’s configuration settings should pop up on the screen.
+   **Note:** If we want to reorganize the order that the stages execute in the pipeline, we can add or remove precursor stages in the Depends On category.
 
-![](/images/Image-2017-03-24-at-4.27.55-PM.png)
+1. In the Deploy Configuration, click on the “Add server group” button. Pick your provider, if more than one is configured. For our example it will be AWS.
 
-Note: If we want to reorganize the order that the stages execute in the pipeline, we can add or remove precursor stages in the Depends On category.
+1. Because this is a new application we will not choose to copy a configuration from a template. Select “Continue without a template”.
```

Comment:
```suggestion
1. Because this is a new application, do not choose to copy a configuration from a template. Press the **Continue without a template** button.
```